packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -1,7 +1,9 @@
 /**
  * External dependencies
  */
-import { deburr, find, words } from 'lodash';
+import { noCase } from 'change-case';
```

Comment:
`noCase` could be overkill given the string being operated on is human search text. Seems like the main thing it does is insert spaces into camel cased text, but I don't expect humans will write that way (even those of us that are coders 😄).

Reply:
Thanks for the feedback - FWIW `_.words()` is pretty costly, and while I haven't directly compared the costs with `noCase()`, I'm almost certain that `noCase()` will be cheaper. Can thoroughly compare if you have doubts, though.

Did you have any other reservations with the changes suggested in this PR, @talldan?